libs/api/domains/document-provider/src/lib/client/models/paperMail.ts

```diff
@@ -0,0 +1,17 @@
+export interface PaperMail {
+  kennitala: string
```

Comment:
Icelandic?

Reply:
Yeah, this service is not generated from a client with a swagger schema. These are the types provided by the service.

This whole section needs a refactor where we remove some of the old stuff and create a new client in line with all the others within the island.is project.